test/e2e/testnets/testnet.go

```diff
@@ -130,21 +130,22 @@ func (t *Testnet) CreateTxClient(name,
 			Msg("error creating txsim")
 		return err
 	}
-	// copy over the keyring directory to the txsim instance
-	err = txsim.Instance.AddFolder(txsimKeyringDir, txsimRootDir, "10001:10001")
+
+	err = txsim.Instance.Commit()
 	if err != nil {
 		log.Err(err).
-			Str("directory", txsimKeyringDir).
 			Str("name", name).
-			Msg("error adding keyring dir to txsim")
+			Msg("error committing txsim")
 		return err
 	}
 
-	err = txsim.Instance.Commit()
+	// copy over the keyring directory to the txsim instance
+	err = txsim.Instance.AddFolder(txsimKeyringDir, txsimRootDir, "10001:10001")
```

Comment:
Maybe just ssh into the txsim instance and check:

- The folder has been mounted correctly
- You can send some transactions using that keyring
- The permissions are correct (not sure the containers which users they use)

If all of this is fine, then I suppose they won't break anything